src/modals/disputeDecline.ts

```diff
@@ -94,18 +101,36 @@ export const execute = async (
 
       const match = botField.value.match(/\((\d+)\)/)
       if (match && match[1] === disputeID) {
+        // If the embed title is "Bot Approved", reject immediately
+        if (embed.title?.toLowerCase() === 'bot approved') {
+          await interaction.editReply({
+            embeds: [
+              errorEmbed(
+                "Can't open ticket",
+                `This bot was approved. If you need help with this bot, please ask in <#714045415707770900> or create a mod ticket above.`
+              ),
+            ],
+          })
+          return
+        }
+
         matchingMessage = msg
         break
       }
     }
 
+    if (
+      fetched.last()?.createdTimestamp &&
+      fetched.last()!.createdTimestamp < cutoff
+    ) {
+      break
+    }
```

Comment:
TypeScript doesn't like the `()!` used here. We could flesh out this line a bit

```suggestion
const lastFetched = fetched.last();
if (lastFetched?.createdTimestamp && lastFetched.createdTimestamp < cutoff) {
    break;
}
```